src/Main.cpp

```diff
@@ -26,14 +28,36 @@ int main()
             sf::Event event;
             while (window.pollEvent(event))
             {
-                if (event.type == sf::Event::Closed)
+              if (event.type == sf::Event::Closed)
                     window.close();
+               
+     
+            switch (event.type)
+            paused = !paused;
+            if (event.type == sf::Event::KeyReleased)
+            {
+                if (event.key.code == sf::Keyboard::Escape);
             }
+                //std::cout << "Esc was pressed" << std::endl;
```

Comment:
This whole section, up to line 55, is unnecessary :) Let's keep this as simple as possible